.flake8

```diff
@@ -10,6 +10,6 @@ ignore = D203,W503
 max-complexity = 16
 max-line-length = 120
 per-file-ignores =
-    **/__init__.py : F401
-    app/main/__init__.py : E402, F403
-    app/status/__init__.py : E402
+    app/__init__.py : F401
```

Comment:
Why the unfavouring of '**'?

Reply:
Because ... it's funny ... flake8 now seems to complain when it's told to ignore an issue that doesn't actually exist in a particular file, and these files now no longer have any error type in common.

Reply:
Weird... they don't have F401 in common?

Reply:
Uhh I can't remember but I think this was the only way I could get it to work. I think actually that the wildcard caused it to match some _other_ `__init__.py` that's not even mentioned here that doesn't have the problem.